pkg/diagnose/runner.go

```diff
@@ -445,6 +448,10 @@ func getSuites(diagCfg diagnosis.Config, deps SuitesDeps) []diagnosis.Suite {
 	catalog.Register("connectivity-datadog-core-endpoints", func() []diagnosis.Diagnosis { return connectivity.Diagnose(diagCfg) })
 	catalog.Register("connectivity-datadog-autodiscovery", connectivity.DiagnoseMetadataAutodiscoveryConnectivity)
 	catalog.Register("connectivity-datadog-event-platform", eventplatformimpl.Diagnose)
+	// port-conflict suite available in darwin only for now
+	if runtime.GOOS == "darwin" {
```

Comment:
Quetsion: How you consider adding some noop port-conflict diagnose for non darwin runtimes, rather than only adding it for darwin?

Reply:
> I guess you could return a single diagnosis saying that it's not implemented

That was what I was suggesting

> it seems better to me to not advertise in that case

Why?

Reply:
I think I agree with Pierre in that I'd rather hide the suite over show it on non-darwin and return a "not implemented yet" message. It seems better to me to hide parts of our code that are not implemented yet, especially since we don't plan on leaving this darwin-specific for very long.